### DISCUSSION TITLE
Next Gen Update Compatibility?

### DISCUSSION THREAD

**Discussion by ZebastianVT** [Aelto/W3_RandomEncounters_Tweaks#158] [Q&A] [answered]
Hi I just want say first thanks for this awesome mod! Its awesome and adds a lot of replayability to the game.

Now on to the topic, CDPR have stated that most mods will be needed to be updated after the Next Gen Patch! I was wondering if you plan on doing so? A updated mod tools will be made few days after launch and they said anything that effects NPCs and etc will be effected. I hope you will Transfer over but if not I respect the decision. Anyways awesome mod and Happy Slaying!

**Comment by Aelto** [comment, marked as answer]
Hello! I'm really happy to hear you're enjoying your time with my mod and glad to see you're hoping for its next gen version!

To answer your question yes I do plan on updating it and as soon as possible. There is a chance all I need updating are scripts, which would make it possible to publish in the first week after the update. And since I don't plan on playing the game without my mod(s) I'll certainly start by updating them! If scripts are not the only thing that needs updating then I'll resort to temporary solutions while the tools get updated.

 Due to some IRL things i won't be able to do it right after the release so I'd suggest checking a day or two after the update, assuming there won't be too many major changes to the code base.

**Reply by mightynumber2** [reply]
So glad to hear this! This is literally my favorite mod for The Witcher 3. Thank you!

**Reply by maribin000** [reply]
You are like Santa, and i'm like a lil kid just waiting for my present every day. Thank you for such an awesome mod!

**Reply by ZebastianVT** [reply]
Fantastic! Great to hear!

**Comment by powerzhea85** [comment]
This is one my fav mods aswell. How can we get notified when you have updated it to the next gen?

**Reply by Aelto** [reply]
Thank you! I think "Watching" the repo and tweaking the watch settings will allow you to get an email as soon as a release is published. You could also star the repository on github to get a link in your github home page, or you can track the mod on nexusmods when i'll update the files.
